recipes/h5coro/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "h5coro" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/h5coro-{{ version }}.tar.gz
+  sha256: 9665cea002b10e3cb262f92d33768a0a116e3233fcc0848db8f81f68db626151
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```

The scripts entry in this package, which prevents it from being noarch, is empty. See https://github.com/ICESat2-SlideRule/h5coro/blob/v0.0.4/setup.py#L19

So it is safe to make it noarch. I would also report that upstream.

```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
```